bitherj/src/main/java/net/bither/bitherj/core/Tx.java

```diff
@@ -867,6 +867,7 @@ public synchronized void signInputs(TransactionSignature.SigHash hashType, Addre
         }
 
         // Every input is now complete.
+        this.recalculateTxHash();
     }
```

Comment:
There is another version of signInputs(), do we need this near line 988 as well?

Reply:
The second signInputs is never called in primer. But for complement, recalculateTxHash is called in both functions.